src/rules/__tests__/no-deprecated-functions.test.ts

```diff
@@ -1,49 +1,230 @@
+import * as fs from 'fs';
```

Comment:
All the `fs` function calls are `await`d on as they're all setup to be drop-in replaced with `{ promises as fs }` when we drop node 7 support.

Reply:
that'll print an experimental warning I think? Maybe not, and if so we can do `util.promisify` manually. Either way, it doesn't matter much in a test I guess

(as a side note, in order to support ESM node has added `import {readFile} from 'fs/promises'` in node 14)

Reply:
I had completely forgotten that warning, good point.

(yeah I noticed they slipped that one into 14.0.0s release - it's an interesting one)